sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -19,42 +20,30 @@
 
 from ._deserialize import _convert_to_entity, _trim_service_metadata
 from ._entity import TableEntity
-from ._error import (
-    _decode_error,
-    _process_table_error,
-    _reprocess_error,
-    _reraise_error,
-    _validate_tablename_error
-)
-from ._generated.models import (
-    SignedIdentifier,
-    TableProperties
-)
+from ._error import _decode_error, _process_table_error, _reprocess_error, _reraise_error, _validate_tablename_error
+from ._generated.models import SignedIdentifier, TableProperties
 from ._serialize import _get_match_headers, _add_entity_properties, _prepare_key
 from ._base_client import parse_connection_str, TablesBaseClient
 from ._serialize import serialize_iso, _parameter_filter_substitution
 from ._deserialize import deserialize_iso, _return_headers_and_deserialized
 from ._table_batch import TableBatchOperations, EntityType, TransactionOperationType
-from ._models import (
-    TableEntityPropertiesPaged,
-    UpdateMode,
-    TableAccessPolicy,
-    TableItem
-)
+from ._models import TableEntityPropertiesPaged, UpdateMode, TableAccessPolicy, TableItem
 
 if TYPE_CHECKING:
     from azure.core.credentials import AzureNamedKeyCredential, AzureSasCredential
 
 
-class TableClient(TablesBaseClient): # pylint: disable=client-accepts-api-version-keyword
+class TableClient(TablesBaseClient):  # pylint: disable=client-accepts-api-version-keyword
```

Comment:
Could we open an issue to fix this pylint disable statement? Our client constructors should have api_version.

Reply:
Logged an issue: https://github.com/Azure/azure-sdk-for-python/issues/28115